metron-deployment/roles/ambari_config/vars/single_node_vm.yml

```diff
@@ -102,7 +102,7 @@ configurations:
 
 required_configurations:
   - metron-env:
-      storm_rest_addr: "{{ groups.ambari_slave[0] }}:8744"
+      storm_rest_addr: "http://{{ groups.ambari_slave[0] }}:8744"
```

Comment:
You'll need to change a ton more than that parameter to enable https. Remember, this file defines a specific cluster. Unsecure with no https. Either http or omitted is fine.